spec/requests/articles_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe "Articles", type: :request do
+  describe "GET article index" do
+    it "returns http success" do
+      get "/articles#index"
```

Comment:
I don't think this is doing what you intend it to do. A URL has a few parts..

protocol://host:port/path

We can also add a `#` to the end or a `?` to the end and then we get more behaviour.

So `/articles#index` will behave the same as `/articles`. This overlaps with the route config suggesting that there is an articles controller with an index action.

I would use `articles_path` here.